From: Ben Hutchings <ben@decadent.org.uk>
Date: Sat, 20 Feb 2016 04:19:41 +0000
Subject: swapops.h: Add #include to fix build failure on powerpc

<linux/swapops.h> uses pte_swp_soft_dirty() and
pte_swp_clear_soft_dirty(), which are defined by <ams/pgtable.h>.  It
appears that in most cases it is either indirectly included or has
previously been included.  However, as of 4.5-rc4 this is not the case
when building kernel/power/swap.c in a 32-bit powerpc configuration.

References: https://buildd.debian.org/status/fetch.php?pkg=linux&arch=powerpc&ver=4.5%7Erc4-1%7Eexp1&stamp=1455791718
Signed-off-by: Ben Hutchings <ben@decadent.org.uk>
---
 include/linux/swapops.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/linux/swapops.h b/include/linux/swapops.h
index 5c3a5f3e7eec..bac77160327f 100644
--- a/include/linux/swapops.h
+++ b/include/linux/swapops.h
@@ -3,6 +3,7 @@
 
 #include <linux/radix-tree.h>
 #include <linux/bug.h>
+#include <asm/pgtable.h>
 
 /*
  * swapcache pages are stored in the swapper_space radix tree.  We want to
